modules/integration/src/test/scala/scala/cli/integration/TestUtil.scala

```diff
@@ -96,6 +96,17 @@ object TestUtil {
     if (uri.startsWith("file:///")) "file:/" + uri.stripPrefix("file:///")
     else uri
 
+  def removeAnsiColors(str: String) = str.replaceAll("\\e\\[[0-9]+m", "")
```

Comment:
NIT I think this'd be even more convenient as a `String` extension method

Reply:
Since, `integration` package is compiled with Scala 2.13 i would rather stick to current state than using implicit class.

Reply:
ah yeah, fair point, forgot about that